packages/geo/src/lib/workspace/shared/workspace.utils.ts

```diff
@@ -0,0 +1,27 @@
+import { WfsWorkspace } from './wfs-workspace';
+import { FeatureWorkspace } from './feature-workspace';
+import { Observable } from 'rxjs';
+import { EntityStoreFilterCustomFuncStrategy, EntityRecord } from '@igo2/common';
+import { map } from 'rxjs/operators';
+import { Feature } from '../../feature/shared/feature.interfaces';
+
+export function mapExtentStrategyActiveToolTip(ws: WfsWorkspace | FeatureWorkspace): Observable<string> {
+  return ws.entityStore.getStrategyOfType(EntityStoreFilterCustomFuncStrategy).active$.pipe(
+    map((active: boolean) => active ? 'igo.geo.workspace.inMapExtent.active.tooltip' : 'igo.geo.workspace.inMapExtent.inactive.tooltip')
+  );
+}
+
+export function FeatureMotionStrategyActiveToolTip(ws: WfsWorkspace | FeatureWorkspace): Observable<string> {
```

Comment:
function name should start with a lowercase.

Reply:
done